lib/db/mysql/index.js

```diff
@@ -27,6 +33,30 @@ function MysqlStore(options) {
     return next();
   };
   this._pool = mysql.createPool(options);
+
+  // Enforce sane defaults on every new connection.
+  // These *should* be set by the database by default, but it's nice
+  // to have an additional layer of protection here.
+  this._pool.on('connection', function(connection) {
```

Comment:
It wasn't obvious to me whether all the various helper methods used `_getConnection` though, it looks like e.g. `_query()` just directly takes a connection from the pool.  Is `_getConnection` intended to replace the pool's default connection-getting logic uniformly?